addons/web/static/tests/views/fields/integer_field.test.js

```diff
@@ -0,0 +1,227 @@
+import { expect, getFixture, test } from "@odoo/hoot";
+import { queryAllTexts, queryOne } from "@odoo/hoot-dom";
+import {
+    clickSave,
+    contains,
+    defineModels,
+    fieldInput,
+    fields,
+    models,
+    mountView,
+    webModels,
+} from "../../web_test_helpers";
+
+class Product extends models.Model {
+    price = fields.Integer({ string: "Price" });
+}
+
+defineModels([Product, webModels.ResUsers, webModels.ResPartner, webModels.ResCompany]);
```

Comment:
webModels.ResUsers, webModels.ResPartner, webModels.ResCompany
This is required for every tests that mount a list view. :( Julien may work on this as it's too much.
Alternatives are: 
- Defining myself a ResUser with a name field.
- mocking the rpc to hasGroup

Reply:
you only have one test with list, just add this at the top of the test `onRpc("/web/dataset/call_kw/res.users/has_group", () => true);`